lookout/style/typos/corrector.py

```diff
@@ -49,10 +49,15 @@ def __init__(self, **kwargs):
         self.ranker = CandidatesRanker()
 
     @property
-    def threads_number(self):
-        """Return the number of threads used to train and to predict."""
+    def threads_number(self) -> int:
+        """Return the number of threads for multiprocessing used to train and to predict."""
         return self.ranker.boost_param["nthread"]
 
+    @threads_number.setter
```

Comment:
I do not see where do you use it. why do you need this setter?

Reply:
It is a way to set number of threads for multiprocessing. It should be provided. And it will definitely be used in upcoming code.

Reply:
just it looks like it is related to logic, but not to a model. 
If you need this number in your Analyser it is better to store it in analyzer config.

Reply:
Well, anyone who uses the model can need to set this number, not only analyzer. It is not the thing that we store, but the thing that is important for running the mode. I still think this method should stay. @vmarkovtsev, could you give your opinion?

Reply:
I don't see anything bad in this setter as long as it is tested. If we see while preparing the release that this code is indeed not used, we are going to delete it anyway.

Reply:
ok, let's keep it.